randovania/games/factorio/gui/ui_files/factorio_game_export_dialog.ui

```diff
@@ -6,73 +6,99 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>527</width>
-    <height>338</height>
+    <width>543</width>
+    <height>351</height>
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Game Patching</string>
+   <string>Factorio Mod Exporter</string>
   </property>
-  <layout class="QGridLayout" name="gridLayout">
-   <item row="2" column="0">
-    <widget class="QLabel" name="output_file_label">
+  <layout class="QGridLayout" name="root_layout">
+   <item row="16" column="0">
+    <widget class="QPushButton" name="accept_button">
      <property name="text">
-      <string>Output Directory</string>
+      <string>Accept</string>
      </property>
     </widget>
    </item>
-   <item row="7" column="0">
-    <widget class="QCheckBox" name="auto_save_spoiler_check">
+   <item row="1" column="0">
+    <widget class="QLabel" name="output_file_label">
      <property name="text">
-      <string>Include a spoiler log on same directory</string>
+      <string>Factorio Mods Folder</string>
      </property>
     </widget>
    </item>
-   <item row="4" column="1">
+   <item row="3" column="1">
     <widget class="QPushButton" name="output_file_button">
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
    </item>
-   <item row="11" column="0">
-    <widget class="QPushButton" name="accept_button">
+   <item row="7" column="0" colspan="2">
+    <widget class="Line" name="download_assets_line">
+     <property name="orientation">
+      <enum>Qt::Orientation::Horizontal</enum>
+     </property>
+    </widget>
+   </item>
+   <item row="4" column="1">
+    <widget class="QPushButton" name="use_default_button">
      <property name="text">
-      <string>Accept</string>
+      <string>Use default</string>
+     </property>
+    </widget>
+   </item>
+   <item row="3" column="0">
+    <widget class="QLineEdit" name="output_file_edit">
+     <property name="placeholderText">
+      <string>Path of the Factorio mod folder</string>
      </property>
     </widget>
    </item>
-   <item row="0" column="0" colspan="2">
-    <widget class="QLabel" name="description_label">
+   <item row="5" column="0" colspan="2">
+    <widget class="QLabel" name="description_b_label">
      <property name="text">
-      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In order to create the randomizer mod, a copy of Factorio itself is needed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If playing from Steam, press &amp;quot;Use default&amp;quot;. On Linux, if using Flatpak Steam the default won't work.&lt;br/&gt;If you downloaded directly from Factorio.com, there will be a mods folder after opening the game once.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
make this a markdown label and then use factorio.com as a hyperlink

Reply:
not really any need. we're not directing people to the site, just identifying it